linchpin/provision/roles/azure/tasks/provision_azure_vm.yml

```diff
@@ -79,7 +79,7 @@
     image: "{{ image | default(omit) }}"
   register: res_def_output
   when: not _async
-  no_log: "{{ not debug_mode }}"
+  # no_log: "{{ not debug_mode }}"
```

Comment:
I assume you don't want to commit this?

Reply:
Yes, I fixed it, I commented it out only for debugging, Thank you!